sleap_client/__init__.py

```diff
@@ -1 +1,2 @@
-from .client import entrypoint
\ No newline at end of file
+from .client import entrypoint
+__version__ = "0.0.1a1"
```

Comment:
you should have a workflow like this so that we have exactly one version per release on pypi. When you update the version you should release or pre-release again. Increment this version before you release again if you already have a pip package on pypi. https://github.com/talmolab/sleap-roots/blob/main/.github/workflows/build.yml